website/pages/docs/installation/postcss.mdx

```diff
@@ -79,34 +79,40 @@ export default defineConfig({
 
 ### Configure the entry CSS with layers
 
-Add this code to an `index.css` file imported in the root component of your project.
+Add this code to an `index.css` file which is going to be the root css of your project.
 
 ```css
 @layer reset, base, tokens, recipes, utilities;
 ```
 
 ### Start your build process
 
-Run your build process with `npm run dev` or whatever command is configured in your package.json file.
+Run your build process by feeding the [root css](#configure-the-entry-css-with-layers) to PostCSS in your preferred way.
 
-<Tabs items={['pnpm', 'npm', 'yarn']}>
+<Tabs items={['CLI', 'JS API']}>
   <Tab>
     ```bash
-    pnpm dev
+    postcss -o output.css index.css
```

Comment:
isn't it postcss-cli ?

Reply:
When you install postcss-cli, it comes with a postcss bin executable